__tests__/__fixtures__/requests/application-form-encoded.js

```diff
@@ -52,7 +52,6 @@ module.exports = {
                 hello: 'world',
               },
               headers: {
-                Accept: '*/*',
```

Comment:
Removed all these `Accept: */*` headers from the response configs because I originally built these response configs from data coming back from a cURL request and cURL adds that `Accept` header to the request (and response since HTTPBin shows incoming headers here) where other targets don't.